packages/aws-cloudfront/README.md

```diff
@@ -51,6 +51,7 @@ AWS_SECRET_ACCESS_KEY=XXX
 distribution:
   component: '@serverless/aws-cloudfront'
   inputs:
+    distributionId: XYZEXAMPLE
```

Comment:
Could we highlight that this is optional? 
Also may be worth documenting somewhere in the README the rules around updating an existing distribution.

Reply:
👍  on it!